block_test.go

```diff
@@ -20,7 +19,9 @@ func TestBlockParsing(t *testing.T) {
 	}
 
 	c := blk.Cid()
-	fmt.Println(c)
+
+	_ = c
+	// f_ = cmt.Println(c)
```

Comment:
It's _legacy_ code. I planned to improve the test on a later PR. Will delete it now.